docs/tutorial/population_dynamics.rst

```diff
@@ -0,0 +1,75 @@
+.. warning::
+    This is early access alpha functionality. While usable, this has not yet been tested in a large number of different
+    scenarios. The API is still be subject to change *without any prior deprecation notice*! If you use this
+    functionality keep a close eye on the changelog for any changes that may affect your analysis.
+
+Population Dynamics
+===================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+
+The following tools enable analysis of experiments which can be described as a system of discrete states. Consider as an example
+a DNA hairpin which can exist in a folded and unfolded state. At equilibrium, the relative populations of the two states are governed by the interconversion kinetics described by the rate constants \\(k_{fold}\\) and \\(k_{unfold}\\)
+
+.. image:: hairpin_kinetics.png
+
+First let's take a look at some example force channel data. We'll downsample the data in order to speed up the caclulations
+and smooth some of the noise::
+
+    raw_force = file.force1x
+    force = raw_force.downsampled_by(78)
+
+    raw_force.plot()
+    force.plot(start=raw_force.start)
+    plt.xlim(0, 3)
+
+.. image:: pop_hairpin_trace.png
+
+The goal of the analyses described below is to label each data point in a state in order to extract
+kinetic and thermodynamic information about the underlying system.
+
+Models
+^^^^^^
+
+Gaussian Mixture Models
+-----------------------
+
+The Gaussian Mixture Model (GMM) is a simple probabilistic model that assumes all data points belong
+to one of a fixed number of states, each of which is described by a (weighted) normal distribution.
+
+We can initialize a model from a force channel as follows::
+
+    gmm = force.train_gmm(2)
+    print(gmm.weights)
+    print(gmm.means)
+
+Here the force data is used to train a 2 component GMM. The component weights and means can be accessed with their respective attributes.
```

Comment:
Maybe briefly mention what "weights" mean in this context (fraction spent in each state).